### PR TITLE
[Frontend] Join US Enhancements #790

### PR DESCRIPTION
**Pull request by SwayamRana808**
## Issue that this pull request solves

 Closes: #790 

## Proposed changes
under admin join us page
1. removed url button
2. linkedIn link working
3. delete button functioning

### Brief description of what is fixed or changed
1. Removed url button from each cards in joinUs page.
2. LinkedIn button which was not redirectly to new tab is now redirecting.
3. Delete button to delete card is now functioning. 
## Types of changes
1.removed url button code.
2. added delete api from backend to delete button from frontend
_Put an `x` in the boxes that apply_

- [ x] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [ x] My code follows the style guidelines of this project
- [ x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ x] My changes generate no new warnings
- [ x ] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Screenshot 2024-05-12 143042

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -1 +1,3 @@
 REACT_APP_API="https://community-website-backend.onrender.com"
+
```
Comment (Kajol-Kumari) [inline frontend/.env:2]: [neat] can you please remove this unnecessary change

File: frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx [hunk 2]
```diff
@@ -13,13 +15,12 @@ export function Card(props) {
...
-                 <div className={style["button-group"]}>
-                    <button className={style["button-edit"]}>URL</button>
-                    <button className={style["button-delete"]}>Delete</button>
+                <a className={style['card-link']} href={props.content.linkdin} target="_blank" rel="noreferrer">LinkedIn</a>
+                 <div className={style["button-group"]}>  
+                    <button className={style["button-delete"]}   onClick={() => props.OnClickDelete(props.content._id)}>Delete</button>
                   </div>
             </div>
-
+
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/Card/Card.jsx:23]: [neat] please make sure the formatting is consistent, i see few extra space here which is un-necessary

File: frontend/src/pages/Admin/Components/JoinUs/JoinUs.jsx [hunk 3]
```diff
@@ -5,10 +5,14 @@ import Grid from "@material-ui/core/Grid"
 import { Card } from './Card/index.js'
 import style from './joinus.module.scss'
 import Loader from "../../../../components/util/Loader";
+// import { useState } from "react";
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/JoinUs.jsx:8]: [neat] please remove these commented code

File: frontend/src/pages/Admin/Components/JoinUs/JoinUs.jsx [hunk 4]
```diff
@@ -21,28 +25,92 @@ export function JoinUs() {
     setIsLoaded(false)
     console.log(data)
   }
+
+    /* card deleting functionality */
+
+  const HandleDeleteSuccess = (id) => {
+    setJoinUsData(prevData => prevData.filter(item => item._id !== id));
+    console.log(id);
```
Comment (Kajol-Kumari) [inline frontend/src/pages/Admin/Components/JoinUs/JoinUs.jsx:33]: [neat] can we please get rid of this console log statement?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@SwayamRana808 The changes looks good to me, please incorporate the requested cleanups and it should be good to be merged :)

There should be a loader while deletion is going on, (take reference from resource deletion page). This will make user aware of the operation going on

**Comment by Kajol-Kumari** [comment]
@SwayamRana808  one more thing i forgot to point above - There should be a loader while deletion is going on, (take reference from resource deletion page). This will make user aware of the operation going on

**Comment by SwayamRana808** [comment]
I have made the changes please review it . 

https://github.com/HITK-TECH-Community/Community-Website/assets/122097724/27c1a7b7-aef3-4637-a4fd-58f05801097a

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍🏻

**Comment by SwayamRana808** [comment]
thank you , this was my first open source contribution ❤️

**Comment by Kajol-Kumari** [comment]
Way more to come, all the best @SwayamRana808 :)
